chris_backend/plugins/models.py

```diff
@@ -240,11 +242,19 @@ def search_name_title_category(self, queryset, name, value):
         lookup = lookup | models.Q(meta__category__icontains=value)
         return queryset.filter(lookup)
 
+
+   def search_name_public_rep(self, queryset, name, value):
```

Comment:
What is the purpose of this function definition?

Reply:
Trying to return any value that contains `public_repo` when you search

Reply:
Where is being called?

Reply:
My understanding is from `pluginMeta table`